### ISSUE TITLE
Friends Timeline Behaves Differently than Public Timeline

### ISSUE THREAD

**Issue by dragonmantank** [spazproject/spazcore#1]
I'm trying to get the Friends Timeline to work based upon the examples/example_timeline.html, which works perfectly fine with the Public timeline. I'm using the newest commit of spazcore as of tonight.
## Changes to Example

I added my twitter username and password to line 69, changed 83 and 84 to:

'success_event':'new_friends_timeline_data',
'failure_event':'error_friends_timeline_data',

and changed line 90 to:

twit.getFriendsTimeline();

I commented out lines 130-140 to cut down on JS errors I was getting with URL expansions.

I then loaded the file in Safari. I tested by adding tweets from a second account to check for updates.
## Expected Result

The Friends timeline would load with the newest tweets at the top like the Public timeline, and every 60 seconds to reload with the newest tweets still at the top.
## Actual Result

With a max_items set to 30, it grabbed the 50 newest tweets and displayed the oldest 30, with the oldest at the top. The timeline would refresh after 60 seconds using the newest tweet for the since_id in the json call (not the 'newest' tweet shown on the screen) but the timeline would not update.

With a max_items set to 50, it grabbed the 50 newest tweets and displays all of them on the screen, oldest to newest. Sometimes it will grab new entries upon a refresh, and sometimes not. 

Leaving max_items at 50, and adding 'add_method': 'append' causes the timeline to render in the correct order (newest to oldest), until a refresh. New entries are appended to the bottom. Sometimes it will append the same 2 or 3 with each refresh.

**Comment by funkatron** [comment]
Yeah, when I was messing with this to get it running on Titanium a little while back, I was seeing issues like this. I didn't have time to delve into why it was happening though. sthg is clearly mucked with my logic tho.
